esmvalcore/preprocessor/_time.py

```diff
@@ -231,6 +213,108 @@ def spans_three_months(time):
     return cube.extract(three_months_bound)
 
 
+def monthly_statistics(cube, operator='mean'):
+    """
+    Compute monthly statistics.
+
+    Chunks time in monthly periods and computes means over them;
+
+    Parameters
+    ----------
+    cube: iris.cube.Cube
+        input cube.
+
+    Returns
+    -------
+    iris.cube.Cube
+        Monthly statistics cube
+    """
+    if not cube.coords('month_number'):
+        iris.coord_categorisation.add_month_number(cube, 'time')
+    if not cube.coords('year'):
+        iris.coord_categorisation.add_year(cube, 'time')
+
+    operator = get_iris_analysis_operation(operator)
+    cube = cube.aggregated_by(['month_number', 'year'], operator)
+    return cube
+
+
+def daily_statistics(cube, operator='mean'):
+    """
+    Compute daily statistics.
+
+    Chunks time in daily periods and computes statistics over them;
+
+    Parameters
+    ----------
+    cube: iris.cube.Cube
+        input cube.
+
```

Comment:
Please mention the supported values for operator for all time statistics function in the docstrings

Reply:
Done